### DISCUSSION TITLE
个别汉字显示不正常

### DISCUSSION THREAD

**Discussion by iTomxy** [sikouhjw/gdutthesis#61] [Q&A] [answered]
写人名的时候用到「璟」字，编译出来显示一个框住的 F：
![hjx](https://user-images.githubusercontent.com/44149820/162229832-92ef67e3-8bc9-4a38-8d3d-ee0c6c126f70.png)
[1] 说是字库的问题，要将 CJK 换成 xeCJK，但我在 [gdutthesis.cls](https://github.com/sikouhjw/gdutthesis/blob/main/gdutthesis.cls) 搜 `CJK` 看到好像已经是在用 xeCJK。
请问这个问题怎么解决？

---

1. https://zhidao.baidu.com/question/873395449540493092.html

**Comment by sikouhjw** [comment, marked as answer]
- 这是 fandol 字体中“缺字”的表现。

- fandol 是 texlive 内置字体，模板将其作为默认字体。

- fandol 缺字比较严重。

- 可根据你的操作系统去选择 `cjk-font` 的选项
  - Overleaf 下可使用 `noto`。

  - Linux 下可考虑安装思源（`Source`/`Noto`）。

  - win/mac 就使用 `windows`/`mac`。

**Comment by Xuezhenggdut** [comment]
![image](https://github.com/sikouhjw/gdutthesis/assets/46556712/80eefa0d-4a48-43f0-8798-4a69a2c126ef)
![image](https://github.com/sikouhjw/gdutthesis/assets/46556712/0a84e6d4-4b56-4e67-9c00-1105d651fcf3)
改了好像没起反应，生僻字祎

**Reply by sikouhjw** [reply]
我觉得你改的位置不是一般思维能想到的。
https://github.com/sikouhjw/gdutthesis/blob/0f3745127922c301296a210dcdcd5de119c00dce/main.tex#L43-L49
